scripts/jenkins/cloud/ansible/roles/ardana_upgrade/tasks/_upgrade-clm.yml

```diff
@@ -0,0 +1,53 @@
+#
+# (c) Copyright 2020 SUSE LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License. You may obtain
+# a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+#
+---
+
+- name: Run ardana-init
+  command: "ardana-init"
+  environment:
+    ARDANA_INIT_AUTO: 1
+
+- name: Copy up input model upgrade script
+  copy:
+    src: "{{ cloud_release }}-input-model-upgrade"
+    dest: "~/cloud-input-model-upgrade"
+    mode: 0755
+  register: ansible_copy_input_model_upgrade
+
+- name: Run input model upgrade script
+  shell: >-
+    ~/cloud-input-model-upgrade
+  register: ansible_input_model_upgrade
```

Comment:
would make sense using the script module here?

https://docs.ansible.com/ansible/2.7/modules/script_module.html

Also, you are saving the script output in a variable but I did not found you using it anywhere.

Reply:
Hadn't encountered the script module before - thanks - very handy.

Reply:
If you don't mind I'll leave switching it to a follow on patch ;-)

Reply:
fine by me